aikau/src/main/resources/alfresco/forms/controls/MaskingTextBox.js

```diff
@@ -155,15 +167,31 @@ define(["dojo/_base/declare",
        */
       postCreate: function alfresco_forms_controls_MaskingTextBox__postCreate() {
          this.inherited(arguments);
-         if (this.mask)
+         if (this.replacements)
          {
-            this._maskRe = new RegExp(this.mask, "g");
+            array.forEach(this.replacements, function(replacement) {
+               if (replacement.regex)
+               {
+                  try
+                  {
+                     var re = new RegExp(replacement.regex, replacement.flags || "g");
```

Comment:
`""` ... if we give them a flags option then ignore it when they specify `""` then that seems a bit counter-intuitive